graphs/possible_bipartition.py

```diff
@@ -1,12 +1,32 @@
 # Can be used for BFS
-from collections import deque 
+from collections import deque
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: the worst, O(n^2)
```

Comment:
Time complexity will be O(E + V) where E is the number of edges and V is the number of vertices/nodes because you are traversing each edge and visiting each vertex once. 

Space complexity is correct. It will be O(n)/ O(V) where n/V are the number of nodes/vertices.